src/icloud.ts

```diff
@@ -11,6 +11,10 @@ function getConstants(): {
 
 export const defaultICloudContainerPath = getConstants().defaultICloudContainerPath
 
+export async function getDefaultICloudContainerPath(): Promise<string | undefined> {
+  return Platform.OS === 'ios' && CloudStore.getDefaultICloudContainerPath();
```

Comment:
Why only iOS? Is the api not available on other platforms?

Reply:
Android does not have this.